quipucords/api/common/report_csv_renderer.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
+"""CSV renderer for reports."""
+
+from api.deployments_report.util import create_deployments_csv
+from api.details_report.util import create_details_csv
+
+from rest_framework import renderers
+
+
+class ReportCSVRenderer(renderers.BaseRenderer):
```

Comment:
Let's split this back out into separate classes.  The utils are really the key for the new API.